index.html

```diff
@@ -0,0 +1,238 @@
+                        <p class="sp-details">
+                            Director of Art Center Nabi and a board member Webtekhy
+                        </p>
+                        <div class="divider"></div>
+                        <p class="footnote">
+                            As the main venue for new media art production in Nigeria. Webtekhy promotes cross-disciplinary collaboration
+                             and understanding among science technologies, humanities, and arts.
+                        </p>
+                    </div>
+                </div>
+                <div class="speaker speaker-4">
+                    <img class="person" src="images/speaker_04.png" alt="The first speaker">
+                    <div class="sp-1-details">
+                        <h3 class="sp-name">Julia Leda</h3>
+                        <p class="sp-details">
+                            Berkman Professor of Entrepreneural Legal Studies at Harvard Law School
+                        </p>
+                        <div class="divider"></div>
+                        <p class="footnote">
+                            Benkler Studies commoms-based peer production, and puublished his seminar book The Wealth of Networks in 2006.
+                        </p>
+                    </div>
+                </div>
+                <div class="speaker speaker-5">
+                    <img class="person" src="images/speaker_05.png" alt="The first speaker">
+                    <div class="sp-1-details">
+                        <h3 class="sp-name">Lilia Tretikov</h3>
+                        <p class="sp-details">
+                            Berkman Professor of Entrepreneural Legal Studies at Harvard Law School
+                        </p>
+                        <div class="divider"></div>
+                        <p class="footnote">
+                            Benkler Studies commoms-based peer production, and puublished his seminar book The Wealth of Networks in 2006.
+                        </p>
+                    </div>
+                </div>
+                <div class="speaker speaker-6">
+                    <img class="person" src="images/speaker_06.png" alt="The first speaker">
+                    <div class="sp-1-details">
+                        <h3 class="sp-name">Ryan Merkley</h3>
+                        <p class="sp-details">
+                            Berkman Professor of Entrepreneural Legal Studies at Harvard Law School
+                        </p>
+                        <div class="divider"></div>
+                        <p class="footnote">
+                            Benkler Studies commoms-based peer production, and puublished his seminar book The Wealth of Networks in 2006.
+                        </p>
+                    </div>
+                </div>-->
+                <div class="more-speakers">
+                    <p class="more">More</p>
+                    <span class="entity">&#5167;</span>
```

Comment:
- [ ] Right now the more button seems to overlap the Speakers as you can see in the image below
> 
![image](https://user-images.githubusercontent.com/90556221/169626142-b5e3ea29-3a9e-4ea3-a386-07c0cf5fd111.png)

Kindly fix this visual bug

Reply:
This is actually working well here on my browser!